README.md

```diff
@@ -9,39 +9,32 @@ To install this cnab bundle, you can use any available cnab runtime. The followi
 ### Steps
 1. Download the latest duffle release for your operating system from duffle's [release page](https://github.com/deislabs/duffle/releases),
  put it somewhere on your path, and make it executable using, for example, `chmod +x /usr/local/bin/duffle`.
-1. Start a kubernetes cluster as described in the riff [getting started](https://projectriff.io/docs/getting-started/) documentation,
-but stop once you've created a cluster and, if appropriate, given yourself cluster-admin permissions.
- If using `minikube`, please run the following command before starting `minikube`:
-    ```
-    minikube config set embed-certs true
-    ```
-1. Check that your kubeconfig file is configured to talk to that cluster:
-    ```
-    kubectl config current-context
-    ```
+1. Start a kubernetes cluster that does not need resources other than kubeconfig for authentication (like minikube OR docker-for-desktop) by following the instructions in [getting started](https://projectriff.io/docs/getting-started/minikube/).
```

Comment:
how about qualifying this with 
> OR a build of duffle which includes the k8s driver auth fixes from https://github.com/deislabs/duffle/pull/786

Reply:
I ended up re-doing the readme for each cluster type.